common/pipes/pubspec.yaml

```diff
@@ -1,10 +1,11 @@
 name: pipes
-description: Angular Dart API Examples for Pipes
+description: AngularDart API Examples for Pipes
 version: 0.0.1
 environment:
   sdk: '>=1.19.0 <2.0.0'
 dependencies:
-  angular2: 2.0.0-beta.22
```

Comment:
In the future, we should constrain the angular2 version more tightly. For example:

```
  angular2: '>=2.2.0 <2.3.0'
```

We should update all of our samples & templates to match (see [web-angular/pubspec.yaml](https://github.com/google/stagehand/blob/master/templates/web-angular/pubspec.yaml) for a good example to copy).